torch/CMakeLists.txt

```diff
@@ -112,10 +112,16 @@ add_custom_command(
   "${TOOLS_PATH}/autograd/templates/python_nn_functions.h"
   "${TOOLS_PATH}/autograd/templates/python_nn_functions_dispatch.h"
   "${TOOLS_PATH}/autograd/templates/variable_factories.h"
-  "${TOOLS_PATH}/autograd/gen_autograd.py"
+  "${TOOLS_PATH}/autograd/deprecated.yaml"
```

Comment:
Does CMake work with the glob `*`? If it does, would it be possible to use that?